BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-99: Form - Super admin

- Add a redux thunk to GET the entity by its ID
- Change some logic to make form work with new getByIdEmployee() thunk
- Implement react-hook-form
- Implement Joi validations and resolver

How to test QA:
1- Navigate to super admins entity in the navbar

2- Test create employee work as intended
2.1- Test new "reset" button, it should reset every field to blank
2.2- Test error messages (check ticket TG-99 Joi validations)

3- Test update employee work as intended
3.1- Test new "reset" button, it should reset every field to the original value
3.2- Test error messages (check ticket TG-99 Joi validations)
